pkg/cacheutil/memcached_client.go

```diff
@@ -297,6 +300,13 @@ func (c *memcachedClient) SetAsync(_ context.Context, key string, value []byte,
 
 		c.duration.WithLabelValues(opSet).Observe(time.Since(start).Seconds())
 	})
+
+	if err == errMemcachedAsyncBufferFull {
+		c.skipped.WithLabelValues(opSet, reasonAsyncBufferFull).Inc()
```

Comment:
Could you initialise it in `newMemcachedClient()` similarly to what we do with `c.skipped.WithLabelValues(opSet, reasonMaxItemSize)`, please?

Reply:
Already did.